scss/base/_generic.scss

```diff
@@ -0,0 +1,19 @@
+* {
+  margin: 0; /* remove all default margins */
+  padding: 0; /* remove all default padding */
+  box-sizing: border-box; /* included border size as part of width value */
+
+  :active {
+    outline: none; /* remove outline from any component that is in active state */
```

Comment:
Is this the right thing to do a11y-wise?

Reply:
> Is this the right thing to do a11y-wise?

Would have thought we would be defining our own. Will add it back in then.